static/templates/settings/organization-settings-admin.handlebars

```diff
@@ -26,16 +26,23 @@
             </div>
 
             <div class="input-group disableable {{#unless realm_allow_message_editing}}control-label-disabled{{/unless}}">
-                <label for="id_realm_message_content_edit_limit_minutes"
-                    id="id_realm_message_content_edit_limit_minutes_label"
-                    title="{{t 'If non-zero, users can edit their message for this many minutes after it is sent. If zero, users can edit all their past messages.' }}">
-                    {{t 'Message edit limit in minutes (0 for no limit)' }}
+                <label for="change_content_edit_limit"
+                    id="change_content_edit_button_label"
+                    title="{{t 'If non-zero, users can edit their message for this many minutes after it is sent. If No Limit (i.e. zero), users can edit all their past messages.' }}">
```

Comment:
This is now considerably clearer, so we can just have the title text be 'Message edit limit" to reduce translation burden.